commands/version_test.go

```diff
@@ -19,11 +22,11 @@ func Test_addVersionDev(t *testing.T) {
 		faasCmd.Execute()
 	})
 
-	if found, err := regexp.MatchString(`(?m:Commit: sha-test)`, stdOut); err != nil || !found {
+	if found, err := regexp.MatchString(`(?m:CLI commit: sha-test)`, stdOut); err != nil || !found {
 		t.Fatalf("Output is not as expected:\n%s", stdOut)
```

Comment:
```
 		t.Fatalf("Output is not as expected:\n%s", stdOut)
```

This is a bad pattern that crept in, we should give the want/got or some other details here.